tests/RuleSet/RuleSetTest.php

```diff
@@ -406,6 +407,25 @@ public static function providePhpUnitTargetVersionHasSetCases()
         }
     }
 
+    private function sortNestedArray(array $array)
+    {
+        foreach ($array as $key => $element) {
+            if (!\is_array($element)) {
+                continue;
+            }
+            $array[$key] = $this->sortNestedArray($element);
+        }
+
+        // sort by key if associative, by values otherwise
+        if (array_keys($array) === range(0, \count($array) - 1)) {
```

Comment:
`range(0, \count($array) - 1)` would that be always the same as `array_values` ?

Reply:
Not for:
```php
$array = [
    'foo' => true,
    'bar' => true,  
];
```